sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/runners/inprocess/AvroIOShardedWriteFactory.java

```diff
@@ -0,0 +1,76 @@
+/*
```

Comment:
Please add inprocess runner specific tests to validate sharding transform application/execution